.circleci/config.yml

```diff
@@ -176,35 +181,34 @@ jobs:
           name: << parameters.test_plugin >>
           no_output_timeout: 10m
           command: |
+              python -c 'import sys; print(sys.version_info[:])'
               export NOX_PYTHON_VERSIONS=<< parameters.py_version >>
               export PLUGINS=<< parameters.test_plugin >>
               conda activate hydra
               pip install nox
               pip install dataclasses
               nox -s lint_plugins test_plugins -ts
-
-
   test_plugin_linux:
     parameters:
       py_version:
         type: string
       test_plugin:
         type: string
     docker:
-      - image: circleci/python:<< parameters.py_version >>
+      - image: circleci/python:3.8
     steps:
       - linux:
           py_version: << parameters.py_version >>
       - run:
           name: << parameters.test_plugin >>
           command: |
-                export NOX_PYTHON_VERSIONS=<< parameters.py_version >>
-                export PLUGINS=<< parameters.test_plugin >>
-                pip install nox
-                pip install dataclasses
-                nox -s lint_plugins test_plugins -ts
-
-
+            export PATH="$HOME/miniconda3/envs/hydra/bin:$PATH"
+            python -c 'import sys; print(sys.version_info[:])'
```

Comment:
The printout does not seem super useful.
remove?

Reply:
done :)